app/src/main/kotlin/app/futured/academyproject/ui/screens/home/HomeViewModel.kt

```diff
@@ -1,20 +1,35 @@
 package app.futured.academyproject.ui.screens.home
 
+import androidx.lifecycle.viewModelScope
+import app.futured.academyproject.domain.GetPlacesUseCase
 import app.futured.academyproject.tools.arch.BaseViewModel
 import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.collections.immutable.PersistentList
+import kotlinx.collections.immutable.toPersistentList
+import timber.log.Timber
 import javax.inject.Inject
 
 @HiltViewModel
 class HomeViewModel @Inject constructor(
     override val viewState: HomeViewState,
+    private val getPlacesUseCase: GetPlacesUseCase
 ) : BaseViewModel<HomeViewState>(), Home.Actions {
 
     init {
         loadCulturalPlaces()
     }
 
     private fun loadCulturalPlaces() {
-        //TODO load somehow data by usecase and pass them to viewstate so they will be shown in UI
+        getPlacesUseCase.execute(){
+            onSuccess {
```

Comment:
Můžeš zkusit Ctrl+Alt+L pro automatické formátování kódu, opraví ti to pár věcí, jinak ok.